yarn-project/cli-wallet/src/storage/wallet_db.ts

```diff
@@ -85,7 +85,7 @@ export class WalletDB {
     await this.#accounts.set(`${address.toString()}:type`, Buffer.from(type));
     await this.#accounts.set(`${address.toString()}:sk`, secretKey.toBuffer());
     await this.#accounts.set(`${address.toString()}:salt`, salt.toBuffer());
-    if (type === 'ecdsasecp256r1ssh' && publicKey) {
+    if ((type === 'ecdsasecp256r1ssh' || type === 'ecdsasecp256r1') && publicKey) {
```

Comment:
mmm, maybe I messed up the names, but this one should behave like `schnorr`, in the sense we store the secret and privateSigningKey.

The SSH one uses the public key as a means of identifying the private key that lives inside the mac.